metabase/metabase
# Convert `spacing.module.css` to CSS Modules

### Description

Converts `spacing.module.css` to CSS modules. These classes handle padding and margins.

### How to verify

These classes touch almost every part of the application, so the best way to review this PR will be to go through the app and just see if things look weird. I've done my best to split up the reviews by selector - if you go through each commit, you'll see that each selector conversion is separated.
